cetmix_tower_server/models/cx_tower_reference_mixin.py

```diff
@@ -186,11 +186,14 @@ def _get_copied_name(self):
         copy_name = _("%(name)s (copy)", name=original_name)
 
         counter = 1
-        while self.search_count([("name", "=", copy_name)]) > 0:
-            counter += 1
-            copy_name = _(
-                "%(name)s (copy %(number)s)", name=original_name, number=str(counter)
-            )
+        if not self.env.context.get("skip_copy_action"):
```

Comment:
Why do we need it?

Reply:
Because our "name" field is not saved, a problem arises when copying "plan line action". The essence of the problem is that the [method ](https://github.com/cetmix/cetmix-tower/blob/fd29f987a08f3eb9ae3f4ce902f1cc969a6f082f/cetmix_tower_server/models/cx_tower_reference_mixin.py#L190)that generates the value for the "reference" field

Reply:
> Because our "name" field is not saved, a problem arises when copying "plan line action". The essence of the problem is that the [method ](https://github.com/cetmix/cetmix-tower/blob/fd29f987a08f3eb9ae3f4ce902f1cc969a6f082f/cetmix_tower_server/models/cx_tower_reference_mixin.py#L190)that generates the value for the "reference" field

In this case you should use more relevant name, because 'skip_copy' is too generic.
Eg 'reference_mixin_skip_copy'

Reply:
Thanks, I'll fix it